Source/Fuse.Reactive/ValueObserver.uno

```diff
@@ -3,6 +3,10 @@ using Uno;
 namespace Fuse.Reactive
 {
 	/** Utility base class that observes the first value of an `IObservable`.
+		
+		Note that this class should only be used with instances that support `IObservable`, 
+		not just `IObservableArray`. This ensures the collection is semantically inteneded
+		for single-value use.
 	*/
 	abstract class ValueObserver: IDisposable, IObserver
```

Comment:
I can't find anything that uses this?

Reply:
It's subclassed and used in several other places.

``` 
$ git grep ValueObserver
Source/Fuse.Reactive.Bindings/Instance.Data.uno:                class ObservableLink: ValueObserver
Source/Fuse.Reactive.Expressions/Subscription.uno:              class ObservableSubscription: ValueObserver
Source/Fuse.Reactive/ValueObserver.uno: abstract class ValueObserver: IDisposable, IObserver
Source/Fuse.Reactive/ValueObserver.uno: class ValueForwarder: ValueObserver
```